README.md

```diff
@@ -58,82 +57,37 @@ Installation will take several minutes.
 The most common paradigm is to promote `master` to `production` by doing a
-fast-forward merge into the branch. This can be accomplished via
-
-```shell
-rake deploy
-```
-
-If someone has had to push emergency changes to the production branch that
-have yet to be merged into master, then you will need "merge down" from
-production into master before doing a deploy. This will ensure that master
-has all the changes pushed to production. *THIS SHOULD NOT HAPPEN NORMALLY.*
-If this has occurred, run
-
-```shell
-rake mergedown
-```
-
-After this, ensure the CI on `master` goes green. Double-check staging as
-you've just introduced a new change to the code. If everything looks good,
-perform a deploy as described earlier to push all changes into production.
+fast-forward merge into the branch. 
 
 ## Development
 
-When developing, you will want one terminal open window running the Jekyll server (see Running the website).
-
 For testing, the following commands are useful:
 
 ```shell
-rake tests:ci  # Rebuilds the website. Runs all tests that the CI system runs.
-rake tests:ci-nobuild  # Same as above but w/o rebuilding for faster iteraion.
-
-rake tests:all  # Rebuilds the website. Runs all tests, including slow ones. Superset of tests:ci
-rake tests:all-nobuild  # Same as above but w/o rebuilding for faster iteration.
+npm run lint:js 	# This runs linter on js files excluding src/js/legacy
+npm run lint:sass	# This runs linter on sass files based on config/sass-lint.yml
+npm run lint		# This will run both js and sass linters
 
-rake tests:htmlproof  # Runs HTML validation as a single-shot.
-rake tests:htmlproof-external-only  # Runs HTML validation of external links only as a single-shot.
+npm run test:e2e	# This runs Nightwatch.js end-to-end tests base on config/nightwatch.js
+npm run test:unit	# This runs  mocha unit tests
+npm run test		# This runs e2e and unit tests
 
-rake tests:javascript  # Runs all javascript tests as a single-shot.
-rake tests:javascript-watch  # Runs all javascript tests continually watching for changes..
 ```
 
-There is currently now way to automatically run `htmlproof` automatically on
-a change to a source file. Patches welcome!
-
-## Updating Karma.
-
-If updating Karma, make sure to remember to rerun `npm shrinkwrap` to update `npm-shrinkwrap.json` (the `npm` equivalent of `Gemfile.lock` for `Bundler`).
-
-## Troubleshooting
-```
-jekyll 2.4.0 | Error:  string not matched
-```
-If you get this error when running `rake serve` it may mean that case sensitivity in your OSX path names are interacting weirdly with Ruby string matching.
-
-For example your path is:
-```
-/Users/awesomeuser/workspace/VA/vets-website/
-```
-but Jekyll is reading it as 
-```
-/Users/awesomeuser/workspace/va/vets-website/
-```
-Changing all the directory names to lower-case should fix it. For some reason, the root level `User` directory does not cause this issue.
+TODO: Add directions about getting the selenium server running to e2e tests.
```

Comment:
you should just be able to run `npm run selenium:bootstrap` and things will all work.